app/components/table-row.js

```diff
@@ -175,10 +176,10 @@ const Row = ({ dat, shareDat }) => (
       <IconContainer className="flex justify-end">
         <FinderButton dat={dat} />
         <LinkButton onClick={() => shareDat(`dat://${dat.key}`)} />
-        <DeleteButton />
+        <DeleteButton onDeleteDat={() => onDeleteDat(dat.key)}/>
```

Comment:
can we refactor this to be

```js
<DeleteButton onClick={() => onDeleteDat(dat.key)}>
```

this way we don't need to touch the `DeleteButton` component

Reply:
I made it work like LinkButton